Makefile

```diff
@@ -65,42 +58,42 @@ gum:
 .PHONY: install-env
 install-env:
 	cp -n .env.sample .env; \
-	ln -sf ${PWD}/.env ${PWD}/packages/web/.env.development; \
-	ln -sf ${PWD}/.env ${PWD}/packages/api/.env; \
-	ln -sf ${PWD}/.env ${PWD}/packages/services/url_shortener/.env; \
-	ln -sf ${PWD}/.env ${PWD}/packages/py-api/.env; \
-	ln -sf ${PWD}/.env ${PWD}/packages/services/questionnaire;
+	ln -sf ${PWD}/.env ${PWD}/services/web/.env.development; \
+	ln -sf ${PWD}/.env ${PWD}/services/url_shortener/.env; \
+	ln -sf ${PWD}/.env ${PWD}/services/py_api/.env; \
+	ln -sf ${PWD}/.env ${PWD}/services/questionnaire;
+
 
 .PHONY: install-python
 install-python:
-	if [ $(shell uname -s) = Linux ]; \
-	then \
-		echo 'export PATH="/$(shell whoami)/.local/bin:$$PATH"' >> ~/.zshrc; \
-		curl -sSL https://install.python-poetry.org | python3 - ;\
-		cd ./packages/py-api && \
-		PATH=/$(shell whoami)/.local/bin:$$PATH poetry install; \
+	if [ $(shell command -v apt-get) ]; then \
+		sudo apt-get update; \
+		sudo apt-get install -y python3.12; \
+		sudo apt install -y pipx; \
+	elif [ $(shell command -v brew) ]; then \
+		brew install python@3.12; \
+		brew install pipx; \
 	else \
-		echo 'export PATH="/Users/$(shell whoami)/.local/bin:$$PATH"' >> ~/.zshrc; \
-		curl -sSL https://install.python-poetry.org | python3 - ;\
-		cd ./packages/py-api && \
-		PATH=/Users/$(shell whoami)/.local/bin:$$PATH poetry install; \
-	fi \
+		echo "No supported package managers found."; \
+		exit 1; \
+	fi; \
+	pipx ensurepath; \
+	pipx install poetry; \
+	poetry config virtualenvs.in-project true; \
+	cd ./services/py_api/; \
+	poetry install; \
+	poetry run pre-commit install;
 
-	echo "Please, reload your shell!"
 
 .PHONY: run-py-api
 run-py-api:
-	cd packages/py-api && poetry run start
+	cd services/py_api && poetry run start
```

Comment:
What's the reason behind the renaming? Usually top level packages especially libs are named with dashes instead of underscores as to avoid confusion between the core code which is initialized with an `init` file and the outer layer which holds all of the config files for managing the environment, etc..

Reply:
Me & Ivan changed the name of the directory to services.